includes/class-recaptcha.php

```diff
@@ -286,6 +298,58 @@ public static function verify_captcha( $captcha_token ) {
 
 		return true;
 	}
+
+	/**
+	 * Hook reCAPTCHA v3 to WooCommerce.
+	 */
+	public static function woocommerce_hooks() {
+		// Add reCAPTCHA v3 to the checkout form.
+		\add_action( 'woocommerce_after_checkout_form', [ __CLASS__, 'add_recaptcha_to_checkout' ] );
+
+		// Verify reCAPTCHA v3 on checkout submission.
+		\add_action( 'woocommerce_checkout_process', [ __CLASS__, 'verify_recaptcha_on_checkout' ] );
+	}
+
+	/**
+	 * Add reCAPTCHA v3 to checkout.
+	 */
+	public static function add_recaptcha_to_checkout() {
+		if ( ! self::can_use_captcha() ) {
+			return;
+		}
+		$settings = self::get_settings();
+		?>
+		<script src="<?php echo \esc_url( self::get_script_url() ); ?>"></script>
+		<script>
```

Comment:
Part of me wants to ask if we should enqueue this as a JS file because of how we were talking the other day about not liking mixing PHP + JS, but this works and I don't see any security concerns about it. :)

Reply:
Being such a small snippet, I'd rather have this approach. The enqueue handling adds more lines of code and I don't think is warranted for such a case.

In this case, we're also adding the script synchronously so we don't have to handle `grecaptcha` not being available when we use it.